examples/.pre-commit-config.yaml

```diff
@@ -0,0 +1,59 @@
+repos:
```

Comment:
Hey @patrickmukumbu , do we need this file in examples?

Reply:
No, let me remove it from here